etc/vagrant/install.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+# Script to set up a Django project on Vagrant.
+
+# Installation settings
+
+PROJECT_NAME=$1
+
+DB_NAME=$PROJECT_NAME
+DB_USERNAME=$PROJECT_NAME
+DB_PASSWD=$PROJECT_NAME
+VIRTUALENV_NAME=$PROJECT_NAME
+
+PROJECT_DIR=/home/vagrant/$PROJECT_NAME
+VIRTUALENV_DIR=/home/vagrant/.virtualenvs/$PROJECT_NAME
+
+
+# Install essential packages from Apt
+apt-get update -y
+# Python dev packages
+apt-get install -y build-essential python python3-dev npm
+# python-setuptools being installed manually
+wget https://bootstrap.pypa.io/ez_setup.py -O - | python3.4
```

Comment:
I just use `pip3 install --upgrade pip` on docker, could we figure out what's a better solution?